### ISSUE TITLE
Add button to download Excel files on class documentation pages

### ISSUE THREAD

**Issue by sujaypatil96** [GenomicsStandardsConsortium/mixs#730]
Add a button on class documentation pages, like, _download XLSX_, somewhere on the page (let's say right above the _Terms_ table).

Upon clicking this button, we should be able to download the XLSX file that is associated with that class.

**Comment by turbomam** [comment]
Identified as low-ish priority by @cmungall and @mslarae13 ?

Please speak up if I got that worng.

**Comment by cmungall** [comment]
Not sure I recall saying that!

If we think that the excel templates are useful to the community (which
surely they are) then I think this is a lightweight and easy way to provide
this. Then also if we want to reference these from a paper or other
documentation we can give a permanent URL rather than a long github URL
which is not guaranteed not to change. Presumably this would be trivial to
implement.

Not sure it should be a button as such. Normal link should be fine.

**Comment by only1chunts** [comment]
I agree that an Excel version of the checklists could be useful to some people.
Please can I clarify what you mean by Excel template though? Looking in the current repo is [this](https://github.com/GenomicsStandardsConsortium/mixs/blob/main/mixs-templates/extensions/Agriculture.xlsx) an example of what you are calling an Excel template? If so, its not worth the effort, just provide a TSV like that. 
Ideally an Excel version would look more like the old style spreadsheet where the names, shortnames, IDs, decriptions etc are included.
I am aware its more effort to do and I agree its not a high priority, but my point is that if the template you are talking of providing is just the top row of names, then its no different to a TSV so its not worth spending time and effort doing it.

**Comment by sformel-usgs** [comment]
I agree that some folks find stick with the excel/tsv format because of its familiarity.  In the long-term, it might be worth developing a MixS version of this DwC tool: https://www.nordatanet.no/aen/template-generator/config%3DDarwin%20Core to give users a GUI to create a template that is useful for their needs. This is not a high priority, but it seemed like a good place/time to share the idea.
